wiki/scripts/build.sh

```diff
@@ -20,8 +20,8 @@ HOST=https://docs.dgraph.io
 # artifact in an appropriate location
 VERSIONS_ARRAY=(
 'v1.1.0'
-'v1.0.17'
 'master'
+'v1.0.17'
```

Comment:
> Place the latest version at the beginning so that version selector can
> append '(latest)' to the version string, and build script can place the
> artifact in an appropriate location

Sure, will be adding a comment about `master` to be placed below the latest version.

Also, I ran the script through `shellcheck`, I will be sending another PR for that.